CHANGELOG

```diff
@@ -1,3 +1,13 @@
+# 1.5.0 (2022-02-07)
+
+ENHANCEMENT:
+
+- Implement Proxy usage ([#11](https://github.com/hashicorp/js-releases/pull/11))
+
+DEPENDENCIES:
+
+- Upgrade openpgpjs to v5.0.1 stable ([#12](https://github.com/hashicorp/js-releases/pull/12))
```

Comment:
We upgraded openpgjs to 5.1.0 in https://github.com/hashicorp/js-releases/pull/15

Reply:
Added two lines since it was technically a code change to upgrade to v5.0.1, whereas 5.1.0 was just a bump